scripts/guess_terminal

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+terminals="roxterm
+sakura
+hyper
+alacrity
+terminator
+termite
+gnome-terminal
+konsole
+xfce4-terminal
+lxterminal
+mate-terminal
+kitty
+yakuake
+tilda
+guake
+eterm
+st
+urxvt
+xterm
+x-terminal-emulator"
```

Comment:
Can we make this a python script that uses our existing guess_terminal() instead of copy/pasting the list?